app/views/schools/dashboard/show.html.erb

```diff
@@ -5,43 +5,85 @@
 <% end %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-three-quarters">
     <span class="govuk-caption-l"><%= @school.name %></span>
     <h1 class="govuk-heading-xl govuk-!-margin-bottom-7">Manage your training</h1>
 
-    <p class="govuk-body-l govuk-!-margin-bottom-5">
-      Check and complete steps for every academic year/cohort
-    </p>
-
-    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">Your cohorts</h2>
-
-    <table class="govuk-table">
-      <thead class="govuk-table__head">
-        <tr class="govuk-table__row">
-          <th class="govuk-table__header" scope="col">Year</th>
-          <th class="govuk-table__header" scope="col">Programme</th>
-          <th class="govuk-table__header" scope="col">Status</th>
-        </tr>
-      </thead>
-      <tbody class="govuk-table__body">
-        <% @school_cohorts.each do |school_cohort| %>
+    <% @school_cohorts.each do |school_cohort| %>
+
+      <% if school_cohort.core_induction_programme? || school_cohort.full_induction_programme? %>
+        <p class="govuk-body govuk-!-margin-bottom-5">
+          <%= govuk_link_to 'Add your early career teacher and mentor details', schools_participants_path(cohort_id: school_cohort.cohort.start_year) %>
+        </p>
+      <% end %>
+
+      <table class="govuk-table">
```

Comment:
Potentially need the NHS responsive table classes here?

Reply:
To test, I added them in as per the `admin/participants/table.html.erb`, with a container and updating the table head and body class to include the NHS responsive classes but couldn't spot any differences, other than the extra padding resulting from the container. Both behaved similarly when resizing. I have removed the width overrides though.